### PR TITLE
Add and fix links

### PR DESCRIPTION
**Pull request by vyrnix9582046173**
- Add reddit social media icon to link to subreddit

- Linked contact to support doc page

- Linked demo button to StartHere doc page

### CODE REVIEW

**Comment by ChristopherTrimboli** [comment]
ok so have you checked to make sure adding new reddit icon works on mobile and ipad too? sometimes adding a new column to a row can throw things off? how does it look?

**Comment by vyrnix9582046173** [comment]
@ChrisEddy Good point, here is what it looks like on mobile and tablet:
screen shot 2018-07-26 at 03 38 22
screen shot 2018-07-26 at 03 37 54

File: src/components/titlePage.js [hunk 1]
```diff
@@ -48,6 +48,10 @@ class TitlePage extends Component {
                             <a href='https://www.youtube.com/channel/UCbeboI7dMETmnUro91AxJZA'>
                                 <i className="fab fa-youtube fa-3x hvr-bob"/>
                             </a>
+                            <a href='https://www.reddit.com/r/exokit/'>
```
Comment (avaer) [inline src/components/titlePage.js]: Not sure if the Reddit link makes sense (yet), since it's not being used.

Comment (vyrnix9582046173) [reply src/components/titlePage.js]: removed reddit icon/link a488d1c

**Review by avaer — APPROVED** [review]
👍

**Pull request merged**